src/Builder/ClassModifier/ClassInstanceModifierStrategy/Applicable/ApplicableInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Integrator\Builder\ClassModifier\ClassInstanceModifierStrategy\Applicable;
+
+use PhpParser\Node\Stmt\ClassMethod;
+
+interface ApplicableInterface
```

Comment:
Maybe we can try to find proper name, e. `ClassInstanceModifierStrategInterface`?

Reply:
Renamed to ApplicableModifierStrategyInterface